content/en/docs/reference/glossary/primary-cluster.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Primary Cluster
+test: n/a
+---
+
+A primary cluster is a cluster with a control plane, which observes all
```

Comment:
Can/should we link to the glossary entry for control plane here?

Reply:
I personally think the term “primary” is too vague. I prefer a name which is a bit more intention revealing such as “control cluster”.

Reply:
I tend to agree "primary" is too vague, and so is "remote cluster" for that matter. Neither term really aligns very well with what they are. A primary cluster is a "self-controlled cluster" and a remote cluster is an "externally-controlled cluster" (the external control can come from the control plane in a self-controlled cluster or a control plane running outside of any cluster). I'm not sure that "self-controlled" and "externally-controlled" are very good names to use, but the point is that "primary" and "remote" don't really seem to be indicative of what they are.